pkg/cri/opts/spec_nonwindows.go

```diff
@@ -0,0 +1,29 @@
+//go:build !windows
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package opts
+
+import (
+	"github.com/containerd/containerd/oci"
+	imagespec "github.com/opencontainers/image-spec/specs-go/v1"
+	runtime "k8s.io/cri-api/pkg/apis/runtime/v1"
+)
+
+func WithProcessCommandLineOrArgs(config *runtime.ContainerConfig, image *imagespec.ImageConfig) oci.SpecOpts {
+	panic("not supported")
```

Comment:
Just make it a no-opt then?

Reply:
Not sure I understand. What do you mean when you say no-opt? Just return nil instead of panic?

Reply:
@dmcgowan ping on this :)

Reply:
Sorry, meant to say no-op. Just return a SpecOpt which doesn't do anything or returns `errdefs.ErrNotImplemented`